### PR TITLE
Attempt to update background colors and remove conditionals

### PR DESCRIPTION
**Pull request by schancel**
This commit attempts to clean up darkmode while updating the color
scheme to not require darkmode conditional. This will make it easier
for us to try various color schemes, or update the pallets in general.

### CODE REVIEW

File: src/components/chat/messages/ChatMessage.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
   <q-item
-    :class="`q-pa-none ${mouseover ? $q.dark.isActive ? 'bg-blue-grey-10' : 'bg-blue-2': $q.dark.isActive ? 'dark' : 'bg-white'}`"
+    :class="`q-pa-none ${mouseover ? $q.dark.isActive ? 'bg-blue-grey-10' : 'bg-blue-2': ''}`"
```
Comment (hlb8122) [inline src/components/chat/messages/ChatMessage.vue:3]: Why does this work?

Comment (oorion) [reply src/components/chat/messages/ChatMessage.vue:3]: Maybe make this a computed property?

File: src/pages/Chat.vue [hunk 2]
```diff
@@ -24,7 +24,7 @@
 
     <q-page-container>
       <q-page>
-        <q-scroll-area ref="chatScroll" class="scroll-area-bordered q-px-none absolute full-width full-height">
+        <q-scroll-area ref="chatScroll" :class="`scroll-area-bordered q-px-none absolute full-width full-height`">
```
Comment (oorion) [inline src/pages/Chat.vue:27]: what does the quote and tick do here?

Comment (schancel) [reply src/pages/Chat.vue]: It executes it as javascript, I'm not sure how this made its way in here though.

**Comment by schancel** [comment]
Alright, this should be good-to-go minus the actual color selection.  This is tbd, but should make it easier to decide upon what we want long term.

**Pull request merged**